src/language-markdown/parser-markdown.js

```diff
@@ -102,6 +103,28 @@ function liquid() {
   };
 }
 
+function abbr() {
```

Comment:
Since there's already a [`remark-abbr`](https://github.com/zestedesavoir/zmarkdown/tree/master/packages/remark-abbr) plugin mentioned in https://github.com/prettier/prettier/issues/5686#issuecomment-450272405, any reason why you don't want to use it here? or is this what you meant "_The contents are left untouched_"? If so, could you open an issue in that repo to suggest this change?

Reply:
I didn't use [remark-abbr](https://github.com/zestedesavoir/zmarkdown/blob/master/packages/remark-abbr/) because it included a [transformer](https://github.com/zestedesavoir/zmarkdown/blob/master/packages/remark-abbr/src/index.js#L34) that replaced the `abbr` in the paragraphs on the fly and I didn't find a clean way to disable it.

*The contents are left untouched* comment was about the fact that there wasn't any additional linting applied to the parsed `node.title`

Reply:
Got it. Could you move those `remark-abbr` code to another file just like what we did for `mdx.js`? though I still think we should ask them to expose the parsing API but this should fine as a temporary workaround.

> _The contents are left untouched_ comment was about the fact that there wasn't any additional linting applied to the parsed `node.title`

I'm not sure what did you mean but if it passed the AST alignment test on CI, it should be fine.